docs/support/ghc-version-support.md

```diff
@@ -18,6 +18,7 @@ Support status (see the support policy below for more details):
 | GHC version  | Last supporting HLS version                                                          | Support status |
 | ------------ | ------------------------------------------------------------------------------------ | -------------- |
 | 9.12.2       | [latest](https://github.com/haskell/haskell-language-server/releases/latest)         | full support   |
+| 9.10.2       | [latest](https://github.com/haskell/haskell-language-server/releases/latest)         | full support   |
```

Comment:
The plugin support table https://haskell-language-server.readthedocs.io/en/stable/support/plugin-support.html needs updating, since the formatters will likely still not work?

Reply:
It's also a bit awkward since it looks like we might never get hlint support for 9.10? or do we have a path to getting that working?

Reply:
The formatters seems to work 9.10.2.